### ISSUE TITLE
Add a dark mode for replay UI

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/ipwb#695]
The web interface is tied to its light display. Many sites have a "dark mode" to account for user preference. ipwb should as well.

In #689, #691, #692, #693, and #694, we made the logo in the README adapt to dark mode, which is ever-present in the GitHub app and potentially in GitHub users' preferences (unverified).

The replay interface will need a background color change (e.g., dark gray -- not black) in addition to the above logo adaptations as potentially text style changes. This might also affect the CSS in the usage of Reconstructive for the archival banners.
